releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,11 @@
 
 [Image][image] | [Sample][sample]
 
+## 2.0.0-preview - 
+
+### Enhancements
+* Added support for [multi-architecture images](https://docs.docker.com/build/building/multi-platform/), the Kubernetes provider can be deployed to `linux/amd` and `linux/arm64` platform. [#892](https://github.com/Azure/AppConfiguration/issues/892)

```

Comment:
The link you referenced says "multi-platform images", not "multi-architecture images". Does this make sense?

Added support for [multi-platform images](https://docs.docker.com/build/building/multi-platform/)**. The** Kubernetes provider **supports** `linux/amd` and `linux/arm64` platform**s**.